packages/nxengine/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://nxengine.sourceforge.net
+TERMUX_PKG_DESCRIPTION="Open-source rewrite engine of the Cave Story for Dingux and MotoMAGX. Author - Caitlin Shaw (rogueeve), httpd://nxengine.sourceforge.net"
```

Comment:
No **long** descriptions. Keep description short, but informative. There no need to keep author and homepage in description, btw the latter one is stored in a separate field `TERMUX_PKG_HOMEPAGE`.